wrench-app/src/main/java/com/izettle/wrench/dialogs/booleanvalue/BooleanValueFragment.kt

```diff
@@ -46,17 +38,17 @@ class BooleanValueFragment : DialogFragment(), Injectable {
         })
 
         return AlertDialog.Builder(requireActivity())
-                .setTitle(R.string.select_scope)
+                .setTitle(".")
```

Comment:
.

Reply:
Turns out you need to define a title upon creation to be able to redefine the text in the title at a later time. In this case using R.string.select_scope felt inappropriate. Is "." less bad than before?
Kinda wanna redo those dialogs to look a little nicer. That will probably involve creating a textview where we can set the name of the configuration. 
Especially the one for "select scope" looks off now.

Reply:
👍 Weird but at least I get why it's a dot now.